### ISSUE TITLE
Compatibility with GNOME 42, and collaboration on app

### ISSUE THREAD

**Issue by jancborchardt** [one-thing-gnome/one-thing#3]
Hi @dantehemerson – so recently @pratap-kumar and I were inspired to do a "One Thing" app for GNOME and must have missed yours, possibly because it is not compatible with GNOME 42.

Do you have any plans on making it compatible?

We have started working on an extension compatible with 42 at https://github.com/one-thing-gnome/one-thing, we are also planning further details like making the One Thing visible on the lock screen.

What do you think about collaborating, do you think that makes sense? Would probably be better than having two apps which do exactly the same in the GNOME extensions store. :D

**Comment by nekohayo** [comment]
I would also like to see this happen, and for the collective you (a merger of the two extensions) to be published on the GNOME Extensions website. I don't need 3 extensions to do the same thing, I want 1 that works, and that solves the workflow usecase/problem I have outlined [over there in this other extension that does the same thing](https://github.com/dukex/most-important-task/issues/12) ;) that is Nextcloud (+ others, ideally) support for me to be able to remotely assign home improvement tasks to my dad.

Note that his computer does not use a lockscreen at all, so it needs to work on the panel and/or the iconless GNOME desktop (both configurable, ideally).

**Comment by dantehemerson** [comment]
Hi @jancborchardt, I just make the extension compatible with version 42 and 43. I recently upgraded Ubuntu 22 and and I noticed that it doesn't work with those versions.

And of course I would like to collaborate to create one extension that implements the funcionalities you are planning. 
I will be checking your version and if I have something to add I will create a PR.

Tell me how we can coordinate to only have one extension published.
